react/components/Animations/Fade.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react'
```

Comment:
This is the same implementation of the [Fade component of modal-layout](https://github.com/vtex-apps/modal-layout/blob/master/react/components/Animations/Fade.tsx). I'm waiting to see another place that I have to implement this animation and implement another type of animation to create a module of animations